tests/unit/general/test_array.cpp

```diff
@@ -14,15 +14,17 @@
 
 using namespace mfem;
 
-TEST_CASE("Array init-list construction", "[Array]")
+TEST_CASE("Array init-list and C-style array constructors", "[Array]")
 {
    int ContigData[6] = {6, 5, 4, 3, 2, 1};
    Array<int> a(ContigData, 6);
-   Array<int> b({6.0, 5.0, 4.0, 3.0, 2.0, 1.0});
```

Comment:
This is no longer possible? If implicit convertibility is kept, it should be, so I would intentionally keep/add this version of the test. The same thing for `Vector` below in the opposite way.

Reply:
This is still possible. I assumed that the `.0` was just a typo, but we can re-add if it was intentional.

Reply:
Maybe was 😆  , but would be nice as an extra test for the convertibility.

Reply:
Re-added in c9cf2ec345f9514506af5da5f8002c5ab14adb21.